### PR TITLE
[annalyns-infiltration]: edited instructions.md for #569

### PR DESCRIPTION
**Pull request by colinleach**
Responding to @ErikSchierboom comments in #569. Is it better than the original?

### CODE REVIEW

**Review by cmcaine — APPROVED** [review]
Thanks!

I made a bunch more changes that I think simplify it further. Let me know what you think!

File: exercises/concept.wip/annalyns-infiltration/.docs/instructions.md [hunk 1]
```diff
@@ -1,24 +1,15 @@
 # Instructions
 
-In this exercise, you'll be implementing the quest logic for a new RPG game a friend is developing. The game's main character is Annalyn, a brave girl with a fierce and loyal pet dog. Unfortunately, disaster strikes, as her best friend was kidnapped while searching for berries in the forest. Annalyn will try to find and free her best friend, optionally taking her dog with her on this quest.
+In this exercise, you'll be implementing the quest logic for a video game a friend is developing. The game's main character is Annalyn, a brave girl with a fierce and loyal pet dog. Unfortunately, disaster strikes, as her best friend Sabīne was kidnapped while searching for berries in the forest. Annalyn will try to find and free Sabīne, optionally taking her dog with her on this quest.
```
Comment (cmcaine) [inline exercises/concept.wip/annalyns-infiltration/.docs/instructions.md:3]:
```suggestion
In this exercise, you'll be writing some logic for a video game a friend is developing. The game's main character is Annalyn, a brave girl with a fierce and loyal pet dog. Unfortunately, disaster strikes, as her best friend Sabīne is kidnapped while searching for berries in the forest. Annalyn will try to find and free Sabīne, optionally taking her dog with her on this quest.
```

File: exercises/concept.wip/annalyns-infiltration/.docs/instructions.md [hunk 2]
```diff
@@ -1,24 +1,15 @@
...
 
-After some time spent following her best friend's trail, she finds the camp in which her best friend is imprisoned. It turns out there are two kidnappers: a mighty knight and a cunning archer.
+Annalyn finds the camp in which Sabīne is imprisoned and it turns out there are two kidnappers: a mighty knight and a cunning archer.
```
Comment (cmcaine) [inline exercises/concept.wip/annalyns-infiltration/.docs/instructions.md:5]:
```suggestion
Annalyn eventually finds the camp in which Sabīne is imprisoned and it turns out there are two kidnappers: a mighty knight and a cunning archer.
The player is presented with some options for what to do next.
For each of the four possible options you need to write a function that tells the game whether it should show that option or not.
```

File: exercises/concept.wip/annalyns-infiltration/.docs/instructions.md [hunk 3]
```diff
@@ -1,24 +1,15 @@
...
 
-Having found the kidnappers, Annalyn considers which of the following actions she can engage in:
+Having found the kidnappers, the player is presented with some options for what to do next.
+For each of the four possible options you need to write a function that tells the game whether it should show that option or not.
```
Comment (cmcaine) [inline exercises/concept.wip/annalyns-infiltration/.docs/instructions.md:9]:
```suggestion
```

File: exercises/concept.wip/annalyns-infiltration/.docs/instructions.md [hunk 4]
```diff
@@ -1,24 +1,15 @@
...
-  - If Annalyn has her pet dog with her she can rescue the prisoner if the archer is asleep.
-    The knight is scared of the dog and the archer will not have time to get ready before Annalyn and the prisoner can escape.
-  - If Annalyn does not have her dog then she and the prisoner must be very sneaky!
-    Annalyn can free the prisoner if the prisoner is awake and the knight and archer are both sleeping, but if the prisoner is sleeping they can't be rescued: the prisoner would be startled by Annalyn's sudden appearance and wake up the knight and archer.
+## 1. Check if the 'Fast Attack' option should be shown
 
-You have four tasks: to implement the logic for determining if the above actions are available based on the state of the three characters found in the forest and whether Annalyn's pet dog is present or not.
-
-## 1. Check if the 'Fast Attack' action is possible
+A fast attack can be made if the knight is sleeping, as it takes time for him to get his armor on, so he will be vulnerable.
```
Comment (cmcaine) [inline exercises/concept.wip/annalyns-infiltration/.docs/instructions.md:12]:
```suggestion
If the knight is sleeping, then Annalyn will be able to make a quick attack into the camp before he can wake up properly and get his armour on.

**Review by cmcaine — APPROVED** [review]

File: exercises/concept.wip/annalyns-infiltration/.docs/instructions.md [hunk 5]
```diff
@@ -29,37 +19,47 @@ julia> can_do_fast_attack(knight_awake)
 false
 ```
 
-_Note: A trailing semicolon suppresses the output of an expression in the top level of the REPL or other interactive contexts (such as Jupyter notebook cells). The output of an expression is only printed by default in the top level of the REPL, not within functions, scripts or other files (unlike what you may expect if you've used Matlab in the past)._ <!-- TODO: Move this to the exercise where the semicolon is first used in examples. Somehow show the Matlab hint only to students who have used Matlab before, when that'll be possible.-->
+<!-- _Note: A trailing semicolon suppresses the output of an expression in the top level of the REPL or other interactive contexts (such as Jupyter notebook cells). The output of an expression is only printed by default in the top level of the REPL, not within functions, scripts or other files._ (unlike what you may expect if you've used Matlab in the past)._ TODO: Move this to the exercise where the semicolon is first used in examples. Somehow show the Matlab hint only to students who have used Matlab before, when that'll be possible.-->
```
Comment (SaschaMann) [inline exercises/concept.wip/annalyns-infiltration/.docs/instructions.md:22]: Why is this commented out?

Comment (colinleach) [reply exercises/concept.wip/annalyns-infiltration/.docs/instructions.md:22]: Seemed a bit confusing and not needed for this exercise. If I read the docs correctly, `instructions.md` should be as simple as possible and extra detail like this should be moved somewhere else. Do you agree this is especially important for an exercise so early in the concepts tree?

Not sure where to move it to, though!

Comment (colinleach) [reply exercises/concept.wip/annalyns-infiltration/.docs/instructions.md:22]: Students new to Julia will probably be in VS Code (or similar) at this point. We don't provide a Jupyter notebook (should we?)

Comment (SaschaMann) [reply exercises/concept.wip/annalyns-infiltration/.docs/instructions.md:22]: And examples may use it, which is arguably the more important reason.

Comment (cmcaine) [reply exercises/concept.wip/annalyns-infiltration/.docs/instructions.md:22]: I hadn't noticed that this had become commented. I think we should make it visible again.

**Pull request merged**